src/en/07_ffi.md

```diff
@@ -471,7 +471,7 @@ environment.
 > ### Recommendation {{#check FFI-NOENUM | Do not use incoming Rust `enum` at FFI boundary}}
 >
 > In a secure Rust development, when interfacing with a foreign language,
-> the Rust code should not accept incoming values from Rust `enum` types.
+> the Rust code should not accept incoming values of type Rust `enum`.
```

Comment:
`enum` is not a Rust type, more like a Rust kind.
Maybe "of any Rust `enum` type"?